features/admin.groups.v1/components/wizard/create-group-wizard-updated.tsx

```diff
@@ -169,7 +174,10 @@ export const CreateGroupWizardUpdated: FunctionComponent<CreateGroupProps> =
             if (isFirstLevelOrganization() || !legacyAuthzRuntime) {
                 getRolesList(null)
                     .then((response: AxiosResponse<RolesV2ResponseInterface>) => {
-                        setRoleList(response?.data?.Resources);
+                        const systemRolesFilteredRolesList: RolesV2Interface[] =
+                            response?.data?.Resources?.filter((role: RolesV2Interface) => !role.meta.systemRole);
```

Comment:
Do the filtering based on extension config

Reply:
Fixed with [bf9b1f1](https://github.com/wso2/identity-apps/pull/6200/commits/bf9b1f1b80a3fedbf206a1fadc20b049641c8df9)